pkgs/top-level/all-packages.nix

```diff
@@ -33395,6 +33395,8 @@ with pkgs;
 
   kubectl-ktop = callPackage ../applications/networking/cluster/kubectl-ktop { };
 
+  kubectl-neat = callPackage ../applications/networking/cluster/kubectl-neat { };
```

Comment:
Please move the package to `pkgs/by-name/ku/kubectl-neat/package.nix` and simply drop this line.

Reply:
:heavy_check_mark: Moved the package. 

Could provide me more context on this change though? New packages shouldn't be added to [pkgs/top-level/all-packages.nix](https://github.com/NixOS/nixpkgs/pull/258395/files#diff-ab5748dc9567516fefba8344056b51ec1866adeace380f46e58a7af3d619ea22) anymore